samples/config/config.sample

```diff
@@ -71,3 +71,34 @@
 # Keys:
 #   camtopic
 #      Gazebo topic where camera images are published.
+#
+#
+# Section [imgcap]:
+#
+# Keys:
+#   width
+#       Width of the image to be captured in Pixels.
+#       Default: Width of the camera sensor.
+#       Possible Values:
+#
+#   height
+#       Height of the image to be captured in Pixels.
+#       Default: Height of the camera sensor.
+#       Possible Values:
+#
+#   format
+#       Image format
+#       Default: 2
+#       Possible Values:
+#            1 - IMAGE_FILE_RAW,  /* Raw Format */
```

Comment:
Are these enum values (e.g. IMAGE_FILE_RAW) ever exposed to a user or developer? If so then showing this value is fine, but you **might** want to put the English version first:
```
#       Possible Values:
#            1 - Raw Format (IMAGE_FILE_RAW)
```

If they are not useful for end users or developers then don't display them:
```
#       Possible Values:
#            1 - Raw Format
```

Reply:
The enum values are declared in CameraParameters.h and used in ImageCapture class. I will edit and put English version first. Also, only jpeg is supported for now. Need a place to mention that.

Reply:
Fixed.

Reply:
Thanks. As per other comment, suggest you just display the one that works - "2". If you wanted to keep the other ones there you could display all of them, but change the "English" version to "Not supported"

so 
```
#       Default: 2 (JPEG)
#       Possible Values:
#            1 - Not supported  (IMAGE_FILE_RAW)
#            2 - JPEG Format (IMAGE_FILE_JPEG)
#            3 - Not supported (...
```

Reply:
Fixed.